src/puppetlabs/trapperkeeper/authorization/ring.clj

```diff
@@ -23,3 +15,36 @@
    schema/Keyword schema/Any})
 
 ;; Functions
+
+(schema/defn get-authorized-authentic? :- schema/Bool
```

Comment:
You mean, could call them `authorized-authentic?` instead of `get-authorized-authentic?`?  I could do that.